nixio/test/test_property.py

```diff
@@ -171,3 +170,53 @@ def test_value_attrs(self):
 
         value.uncertainty = 0.5
         assert(value.uncertainty == 0.5)
+
+
+@unittest.skipIf(skip_cpp, "HDF5 backend not available.")
+class TestPropertyCPP(_TestProperty):
+
+    def setUp(self):
+        self.file    = File.open("unittest.h5", FileMode.Overwrite,
+                                 backend="hdf5")
```

Comment:
I was also thinking that the test superclasses could define the setUp method and the subclasses could just define their own backend in a static variable, used in the superclass setUp method.